distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1608,17 +1608,18 @@ type Table
                 column_type_suggestions = preprocessed.internal_columns.map .sql_type_reference
                 self.connection.read_statement sql column_type_suggestions
 
-    ## UNSTABLE
+    ## PRIVATE
+       Creates a query corresponding to this table.
+    to_select_query : Query
+    to_select_query self =
+        cols = self.internal_columns.map (c -> [c.name, c.expression])
+        # assert cols.not_empty
```

Comment:
Is this interpreted by the compiler or just a regular comment?

Reply:
Regular comment.

We have plans to support Enso assertions #5973 so I thought it's worth to mark these so that we can then grep and replace them with proper thing, once available.

Reply:
That's a great idea, I'll start doing that too.